lib/ditcher.js

```diff
@@ -49,9 +50,13 @@ var crit_ops = {
   eq: function (query, fields) {
     var field;
     if (null !== fields) {
-      for (field in  fields) {
+      for (field in fields) {
         if (fields.hasOwnProperty(field)) {
-          query[field] = fields[field];
+          if (fields[field].type === "Any") {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ANestedIfDepth)